### DISCUSSION TITLE
Tuning parameters for the trunk.

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#275] [TF2: Tuning]
This is the version of the tuning we would like to put on the trunk for LT and MT

Tuning (this is the so-called `Tuning F`): 
```
clubb_l_predict_upwp_vpwp=.true. 
clubb_l_mono_flux_lim_um   = .true.  
clubb_l_mono_flux_lim_vm   = .true.  
clubb_c_uu_shr = 0.1
clubb_c7=0.1

micro_mg_dcs=0.00025
micro_mg_vtrmi_factor=0.5
microp_aero_wsubi_scale=2.5

dust_emis_fact         = 0.80D0
seasalt_emis_scale=1.5
```

@cacraigucar, @JulioTBacmeister, @bstephens82

**Comment by bstephens82** [comment]
I think that looks right from what I understand.

**Comment by cacraigucar** [comment]
Is this ready for me to incorporate into a CAM tag?  Are these settings we want to default for all cam_dev runs as well?  Adding @adamrher to this discussion to get his input on these settings for all cam_dev.

**Reply by adamrher** [reply]
fine by me for these to apply to all cam_dev cases. These are subject to change as we couple to the ocean, but it would be convenient at this stage to get this basic template on the trunk that we can then fine tune later.
